packages/calcite-components/src/components/scrim/scrim.scss

```diff
@@ -36,7 +39,7 @@
   justify-center
   overflow-hidden;
   animation: calcite-scrim-fade-in var(--calcite-internal-animation-timing-medium) ease-in-out;
-  background-color: var(--calcite-scrim-background, var(--calcite-scrim-background-internal));
+  background-color: var(--calcite-scrim-background, var(--calcite-color-background-transparent-scrim));
```

Comment:
I _think_ this is no longer needed since it's `--calcite-scrim-background` is now defined at the `:host` level. We can remove afterwards if this is the case. cc @driskull @macandcheese

Reply:
part of the errors I was running into came when I tried to remove this and a few other tokens that I thought were no longer needed. We can do cleanup later